analyses/snv-callers/README.md

```diff
@@ -1,213 +1,75 @@
+This file is MAF-like meaning it is TSV file that contains many of the fields of a [MAF file](https://docs.gdc.cancer.gov/Data/File_Formats/MAF_Format/) but also some added calculations like [Variant Allele Fraction](#variant-allele-fraction-calculation) and some sample metadata information.
+These files combine the [MAF file data](https://docs.gdc.cancer.gov/Data/File_Formats/MAF_Format/) from 3 different SNV callers: [Mutect2](https://software.broadinstitute.org/cancer/cga/mutect), [Strelka2](https://github.com/Illumina/strelka), and [Lancet](https://github.com/nygenome/lancet).
+See the methods on the callers' settings [here](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/03.methods.md#somatic-single-nucleotide-variant-calling) and see [the methods of this caller analysis and comparison below](#summary-of-methods).
 
-### 04-create_consensus_mut_files.R
-
-This script takes the merged output of 03-merge_callers.R and saves the final consensus mutation calls to a MAF-like file.
-This script outputs two main files:  
-- `consensus_mutation_vaf.tsv` - contains the consensus mutations and their associated variables.
-- `consensus_mutation_tmb.tsv` - contains the re-calculated TMBs based on the conensus mutations only.
-
-**Option descriptions**
-```
- --merged_files : File path to where the 03-merged_callers.R output can be found.
-                  Files required: "all_callers_vaf.<file_format>"
-                                  "all_callers_tmb.<file_format>"
-                                  "mutation_id_list.<file_format>"
- --vaf : What VAF should be used when combining callers? Options are 'median' or
-         one of the caller names."
- --combo : What combination of callers need to detect a mutation for it to be
-           considered real and placed in the consensus file? List the callers
-           that need to be considered in alphabetical order with '-'
-           in between. eg. 'lancet-mutect2-strelka2'
- --file_format: What type of file format were the vaf and tmb files saved as? Options are
-               "rds" or "tsv". Default is "rds".
- --output : Where you would like the output from this script to be stored.
- --bed_wgs : File path that specifies the caller-specific BED regions file.
-             Assumes from top directory, 'OpenPBTA-analysis'.
- --bed_wxs : File path that specifies the WXS BED regions file. Assumes file path
-             is given from top directory of 'OpenPBTA-analysis'
- --overwrite : If TRUE, will overwrite any reports of the same name. Default is
-             FALSE
-```
-
-# Individual Caller Evaluation
-
-The first step in this analysis is an individual evaluation of each MAF from each caller.
-This analysis prints out a report that is further subdivided by the WGS and WXS samples.
+It is "MAF-like" file because it has many of the same fields as a MAF file but..  
```

Comment:
Okay, this the third time we define MAF-like... we should be less repetitive.

Reply:
Fair enough! ha.